### DISCUSSION TITLE
Possible to disable "old style DOS executable MZ" loader for Windows binaries?

### DISCUSSION THREAD

**Discussion by clearbluejar** [NationalSecurityAgency/ghidra#6343] [Q&A] [answered]
When importing most (any?) modern Windows binary two loaders are triggered. 

<img width="772" alt="image" src="https://github.com/NationalSecurityAgency/ghidra/assets/3752074/f711bd05-ba06-42af-a206-8ed7eb22f28d">

Each time I need to uncheck "old-style" so that I don't have duplicates in the project.

<img width="313" alt="image" src="https://github.com/NationalSecurityAgency/ghidra/assets/3752074/be2a4ce4-82f1-4a37-b382-0d9a3735c3d4">

Is there a way to disable the old style loader?

**Comment by astrelsky** [comment]
Uncheck them

**Comment by clearbluejar** [comment]
I can uncheck them, but  don't want to have to do it every time I load a new binary. Is there was a standard way to disable the loader?

**Comment by astrelsky** [comment]
This menu should only be seen when doing a batch import and not when just importing one file.

**Comment by ryanmkurtz** [comment]
This is just how Batch Import works.  If you selected a directory/container that contains files that Ghidra can import, it's going to assume you want to import them by default.

**Comment by clearbluejar** [comment]
If I am often using batch import for Windows files, and don't want to use the "old style DOS executable importer", is there a way to disable just that loader?

**Reply by ryanmkurtz** [reply, marked as answer]
No, not without modifying the source.
